SwiftWeather/WeatherViewController.swift

```diff
@@ -9,113 +9,116 @@ import MobileCoreServices
 
 //MARK: - UIViewController Properties
 class WeatherViewController: UIViewController {
-
-  //MARK: - IBOutlets
-  @IBOutlet weak var locationLabel: UILabel!
-  @IBOutlet weak var iconLabel: UILabel!
-  @IBOutlet weak var temperatureLabel: UILabel!
-  @IBOutlet var forecastViews: [ForecastView]!
-
-  let identifier = "WeatherIdentifier"
-  
-    //MARK: - Super Methods
-  override func viewDidLoad() {
-    super.viewDidLoad()
-    viewModel = WeatherViewModel()
-    viewModel?.startLocationService()
-    setA11yIdentifiers()
-  }
-
-  override func viewWillAppear(_ animated: Bool) {
-    super.viewWillAppear(animated)
-    
-    locationLabel.center.x  -= view.bounds.width
-    iconLabel.center.x -= view.bounds.width
-    temperatureLabel.center.x -= view.bounds.width
-    
-    iconLabel.alpha = 0.0
-    locationLabel.alpha = 0.0
-    temperatureLabel.alpha = 0.0
-  }
-
-  override func viewDidAppear(_ animated: Bool) {
-    super.viewDidAppear(animated)
-    
-    UIView.animate(withDuration: 0.5, animations: {
-     self.locationLabel.center.x += self.view.bounds.width
-    })
     
-    UIView.animate(withDuration: 0.5, delay: 0.3, usingSpringWithDamping: 0.2, initialSpringVelocity: 0.0, options: [], animations: {
-     self.iconLabel.center.x += self.view.bounds.width
-    }, completion: nil)
+    //MARK: - IBOutlets
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var iconLabel: UILabel!
+    @IBOutlet weak var temperatureLabel: UILabel!
+    @IBOutlet var forecastViews: [ForecastView]!
     
-    UIView.animate(withDuration: 0.5, delay: 0.4, usingSpringWithDamping: 0.2, initialSpringVelocity: 0.0, options: [], animations: {
-     self.temperatureLabel.center.x += self.view.bounds.width
-    }, completion: nil)
+    let identifier = "WeatherIdentifier"
     
+    //MARK: - Super Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        viewModel = WeatherViewModel()
+        viewModel?.startLocationService()
+        setA11yIdentifiers()
```

Comment:
I prefer to have the full name like `setAccessibilityIdentifiers()` 🤣  but the old name was from you 😉

Reply:
mmh I did not change that name, just moved it into a new function jeje, but that Is true I must rename it!!

Reply:
@saulurias definitely was NOT from you, thanks for fixing it

Reply:
Woops, did not read that "NOT" jaja

Reply:
@saulurias was my fault, I added the *NOT* bit later on 😅 you can see the history by clicking on the *edited* link.